pkg/app/pipectl/cmd/quickstart/quickstart.go

```diff
@@ -29,11 +29,10 @@ import (
 	"time"
 
 	"github.com/manifoldco/promptui"
-	"github.com/spf13/cobra"
-
 	"github.com/pipe-cd/pipecd/pkg/backoff"
 	"github.com/pipe-cd/pipecd/pkg/cli"
 	"github.com/pipe-cd/pipecd/pkg/version"
+	"github.com/spf13/cobra"
```

Comment:
[nit] For convention, should add this line to the above block 🙏 

The import lines are ordered like this ↓
1. standard library
2. Third-party library
3. our library (packages from this repo)